File: Length of Last Word/length_of_last_word.py
# LeetCode Submissions by Xinyu Chen
# Length of Last Word
# https://leetcode.com/problems/length-of-last-word/
# Runtime: 54 ms

class Solution:
    # @param {string} s
    # @return {integer}
    def lengthOfLastWord(self, s):
        words = s.split()
        if len(words) == 0:
            return 0
        else:
            return len(words[-1])

File: Length of Last Word/length_of_last_word.java
/*	
 *	LeetCode Submissions by Xinyu Chen
 *	Length of Last Word
 *	https://leetcode.com/problems/length-of-last-word/
 *	Runtime: 250 ms
 */

public class Solution {
    public int lengthOfLastWord(String s) {
        String[] words = s.split(" ");
        if (words.length == 0) return 0;
        return words[words.length - 1].length();
    }
}